sdk/spring/azure-spring-boot-starter-servicebus-jms/pom.xml

```diff
@@ -40,7 +40,42 @@
     <dependency>
       <groupId>org.apache.qpid</groupId>
       <artifactId>qpid-jms-client</artifactId>
-      <version>0.53.0</version><!-- {x-version-update;org.apache.qpid:qpid-jms-client;external_dependency} -->
+      <version>0.53.0</version> <!-- {x-version-update;org.apache.qpid:qpid-jms-client;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-buffer</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-buffer;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-common</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-common;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-handler</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-handler;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-transport;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-transport-native-epoll;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-kqueue</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-transport-native-kqueue;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-codec-http;external_dependency} -->
```

Comment:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/scripts/managed_external_dependencies.txt#L868-L898

Do we need to add these all?

Reply:
I only added the netty module that qpid depends on, in order to solve the problem that the netty version that qpid depends on may be inconsistent with the SDK.